src/test/kotlin/it/pagopa/wallet/services/WalletEventSinksServiceTest.kt

```diff
@@ -0,0 +1,98 @@
+package it.pagopa.wallet.services
+
+import it.pagopa.wallet.WalletTestUtils.WALLET_DOMAIN
+import it.pagopa.wallet.audit.LoggedAction
+import it.pagopa.wallet.audit.LoggingEvent
+import it.pagopa.wallet.audit.WalletAddedEvent
+import it.pagopa.wallet.config.properties.RetrySavePolicyConfig
+import it.pagopa.wallet.repositories.LoggingEventRepository
+import java.time.Duration
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.*
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Sinks
+import reactor.test.StepVerifier
+
+class WalletEventSinksServiceTest {
+    private var loggingEventRepository: LoggingEventRepository = mock()
+    private var retrySavePolicyConfig: RetrySavePolicyConfig = RetrySavePolicyConfig(1, 1)
+    private val walletEventSink: Sinks.Many<LoggedAction<*>> = mock()
+    private lateinit var walletEventSinksService: WalletEventSinksService
```

Comment:
```suggestion
    private val walletEventSinksService = WalletEventSinksService(loggingEventRepository, retrySavePolicyConfig, walletEventSink)
```
initialize walletEventSinksService once here instead of duplicating it

Reply:
Done in [66ae585](https://github.com/pagopa/pagopa-wallet-service/pull/195/commits/66ae5854b35d5091f5cc0057ea5d03e35b50a0f9)